templates/java/TestingFramework/README.md

```diff
@@ -68,14 +68,34 @@ Prerequisites:
  * Have java installed
  * Have maven installed
  * Build the template `mvn clean verify`
- * Download the latest nucleus `curl -s https://d2s8p88vqu9w66.cloudfront.net/releases/greengrass-nucleus-latest.zip > greengrass-nucleus-latest.zip`
-   * Note down the path of the zip in your file system (you'll need it below)
+ * Download the latest nucleus
+    * On Linux Distros `curl -s https://d2s8p88vqu9w66.cloudfront.net/releases/greengrass-nucleus-latest.zip > greengrass-nucleus-latest.zip`
+    * On Windows Powershell `iwr -Uri "https://d2s8p88vqu9w66.cloudfront.net/releases/greengrass-nucleus-latest.zip" -OutFile "greengrass-nucleus-latest.zip"`
+* Note down the path of the zip in your file system (you'll need it below)
 
 
 From the root of the project run (make sure to replace the values)
 
+**On Linux Distros**
+
 `java -Dggc.archive=<path-to-nucleus-zip> -Dtest.log.path=./results -Dtags=Sample -jar ./uat/target/ComponentUATs-1.0.0-SNAPSHOT.jar`
 
+**On Windows**
+
+Before you can run the tests on windows you first have to Configure user credentials for Windows devices
+
+* Download the [psexec utility](https://learn.microsoft.com/en-us/sysinternals/downloads/psexec)
+
+* Create user in the LocalSystem account on the Windows device. Run the commands below and replace user-name with the same name of your current system user (output from the previous command) and password with a secure password.
+    * Run  `net user user-name password /ADD`
+    * Run `psexec -s cmd /c cmdkey /generic:user-name /user:user-name /pass:password`
```

Comment:
this shouldn't be needed for OTF to work. OTF does not run the Nucleus as the SYSTEM user unless I'm mistaken.

Reply:
Well I had to do this for it to work and I am using otf as a dependency on and ec2 windows machine